src/PSR7/Exception/InvalidRequestMessage.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace League\OpenAPIValidation\PSR7\Exception;
+
+use Exception;
+use Throwable;
+
+class InvalidRequestMessage extends Exception
```

Comment:
Btw, @scaytrase should we extend from ValidationFailed as well here?

Reply:
Actually we can. Do not see any problems with it now